### DISCUSSION TITLE
Welcome to holbertonschool-low_level_programming Discussions!

### DISCUSSION THREAD

**Discussion by Miguel22247** [Miguel22247/holbertonschool-low_level_programming#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by parzival-p1** [comment]
Hello everyone! This is Francisco Hernández (aka Paco), I'm a newbie from Holberton School Mexico! 🇲🇽. 

Right now Mexico City's campus it's just online so I'd like to find some peers/mentors to collaborate with for Holberton's projects, sometimes it's hard to solve the projects by myself. 

PD. Hope this discussion still working  😁
